runtime/Cargo.toml

```diff
@@ -33,45 +33,50 @@ url = "2"
 cfg-if = "1.0"
 prometheus = { version = "0.12.0", features = ["process"] }
 lazy_static = "1.4.0"
+base58 = { version = "0.2.0" }
+blake2 = { version = "0.10.4", default-features = false }
+scale-decode = { version = "0.7.0",  features = ["derive"] }
+scale-encode = { version = "0.3.0",  features = ["derive"] }
 
 # Substrate dependencies
 sp-arithmetic = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.40" }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.40", default-features = false }
 # needs to agree with subxt's version
 sp-keyring = { version = "7.0.0", default_features = false }
 sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.40" }
 frame-support = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.40" }
 
 # Subxt dependencies
-subxt = "0.25.0"
+subxt = { version = "0.29.0", default_features = false,  features = ["jsonrpsee-ws"] }
```

Comment:
Is `default_features = false` required?

Reply:
mentioned in the [above comment](https://github.com/interlay/interbtc-clients/pull/483#discussion_r1259189848)